src/login_helper.js

```diff
@@ -61,6 +66,347 @@ function filterPastAndUpcomingTrips(travelerTrips) {
+  let agentFee = costBeforeAgentFee * 0.1;
+  let totalTripCost = costBeforeAgentFee + agentFee;
+  costs.push(
+    flightCosts,
+    lodgingCostsPerDay,
+    totalLodgingCosts,
+    costBeforeAgentFee,
+    agentFee,
+    totalTripCost
+  );
+  return costs;
+}
+
+function renderDestinationName(destination) {
+  let destinationName = document.createTextNode(
+    `Trip Destination: ${destination.destination}`
+  );
+  let destinationNameParagraph = document.createElement("paragraph");
+  destinationNameParagraph.appendChild(destinationName);
+  document.querySelectorAll("section")[3].appendChild(destinationNameParagraph);
+}
+
+function renderNumberOfTravelers() {
+  let values = getValuesFromInputs();
+
+  let numOfTravelersNode = document.createTextNode(
+    `Number of Travelers: ${values[3]}`
+  );
+  let numOfTravelersParagraph = document.createElement("paragraph");
+  numOfTravelersParagraph.appendChild(numOfTravelersNode);
+  document.querySelectorAll("section")[3].appendChild(numOfTravelersParagraph);
+}
+
+function renderStartandEndDate(tripRequestStartDate, tripRequestEndDate) {
+  let formattedStartDate = new Date(tripRequestStartDate);
+  formattedStartDate = new Date(
+    formattedStartDate.getTime() +
+    formattedStartDate.getTimezoneOffset() * 60000
+  );
+  let formattedEndDate = new Date(tripRequestEndDate);
+  formattedEndDate = new Date(
+    formattedEndDate.getTime() + formattedEndDate.getTimezoneOffset() * 60000
+  );
+  let startDateParagraph = document.createElement("paragraph");
+  let endDateParagraph = document.createElement("paragraph");
+  let startDateNode = document.createTextNode(
+    `Trip Start Date: ${formattedStartDate.toDateString()}`
+  );
+  let endDateNode = document.createTextNode(
+    `Trip End Date: ${formattedEndDate.toDateString()}`
+  );
+  startDateParagraph.appendChild(startDateNode);
+  endDateParagraph.appendChild(endDateNode);
+  document.querySelectorAll("section")[3].appendChild(startDateParagraph);
+  document.querySelectorAll("section")[3].appendChild(endDateParagraph);
+}
+
+function renderTripCost(destination, tripRequestData, allTrips) {
+  let costs = calculateTripRequestCosts(destination, tripRequestData, allTrips);
+  let totalCostNode = document.createTextNode(
+    `Total Estimated Trip Cost: $${costs[5].toFixed(2)}`
+  );
+  document.querySelectorAll("section")[3].appendChild(totalCostNode);
+}
+
+function confirmAndPostTrip(destination, tripRequestData, allTrips) {
+  getSingleTraveler(destination, tripRequestData, allTrips).then(function (
+    result
+  ) {
+    let values = getValuesFromInputs();
+    let tripRequestStartDate = values[1];
+    let tripRequestEndDate = values[2];
+    let traveler = result;
+    let userId = traveler.traveler.id;
+    let tripRequestId = allTrips.length + 1;
+    const data = JSON.stringify({
+      id: parseFloat(tripRequestId),
+      userID: userId,
+      destinationID: tripRequestData[0],
```

Comment:
hardcoding these array indexes is not a great way to manipulate data...which is why i suggest turning the output of that getValuesFromInputs(); method to an object.  Makes for much cleaner manipulation

Reply:
try to eliminate all hard coded indexes by working with objects instead, and using the names of its attributes